### DISCUSSION TITLE
Recomiendan Actulizar de Windows 10 al 11?

### DISCUSSION THREAD

**Discussion by JorgeCanto3** [LaunchX-InnovaccionVirtual/CursoIntroPython#141] [Q&A]
Tengo esta duda de si sea mejor actualizarlo o esperar por algunos bugs que pueda traer la nueva version.

Saludos ヾ(￣▽￣) Bye~Bye~

**Comment by IdeIsaac** [comment]
He visto las recomendaciones en internet y dicen que aún le falta, quizás esperes mejor a mediados de años para estar más seguros

**Comment by OscarDelgad0** [comment]
Que tal, dependerá de tu equipo y tu forma de manejarlo. Actualizar tu sistema operativo puede tener ventajas y desventajas, al ser un sistema nuevo, puede traer errores y problemas de compatibilidad. En mi opinión, preferiría mantenerme en windows 10, y esperar un poco mas para pasar a windows 11.

**Comment by enriqueorea** [comment]
Actualmente no existen ventajas al actualizar, en lo personal no valdría la pena

**Comment by DanielOe21** [comment]
# Opinión de usuario 'arriba del promedio'

Hola! En lo personal uso **W10**, he sabido de buenas experiencias con W11 de otras personas, incluso nadie cercano me ha comentado sobre algún bug 🐞 o nada parecido.

Yo me considero un usuario ligeramente arriba del promedio, considerando que la gran mayoría de las personas sólo usan su equipo para navegar en internet, ver netflix y usar Word o Excel... En mi caso voy un poquito más allá del uso normal, ya que manejo algunos programas un poquito más específicos de mi área y me gusta manejar la compu 💻 a un nivel medio personalizado, pero nada fuera de los parámetros de un simple mortal.

Te decía, en lo personal me gusta esperar un tiempo ⏱ para actualizar, ya que aunque W11 esté bastante probado antes de lanzarlo al mercado, nunca estamos exentos, no sé si te enteraste de hecho de que hubo un problema con los procesadores de AMD por ejemplo... Además, es muy normal que algunos drivers, programas o aplicaciones tarden en actualizarse para funcionar correctamente 🤓.

Mi recomendación personal es que, si te funciona W10 por ahora, no le muevas. No he encontrado de momento alguna razón significativa como para actualizar a W11 más que si acaso la novedad 😋

Por favor no te olvides de elegir una respuesta a tu pregunta y marcarla como tal. Salu✌

**Reply by mgc14** [reply]
Yo llevo un mes con W11 y la verdad soy feliz.

Me quitaron muchos problemas que tenía de bugs de Windows 10.

**Comment by irvingfloley** [comment]
Si eres desarrollador te recomendaría que te esperes, justamente nuestro equipo tiene problemas usando Windows 11 y Visual Studio 2022, bajo cierta condición el debugger usando la clase DateTime de .NET genera errores al momento de querer conocer el valor de una variable, este error solo sucede con esa combinación, en otros equipos con Visual Studio 2022 no sucede, solo en la que tiene Windows 11. 

De momento no hemos podido corregir el detalle, o buscar información de otros usuarios con problemas similares y su solución en Internet, también hemos experimentado problemas con algunos software para VPN y conexión a escritorio remoto en servidores cuyas versiones de Windows Server son antiguas, cosa que no sucede en W10.

Espero esta información te ayude a tomar una decisión.

**Comment by RafBD** [comment]
Yo utilizo Windows 11 y hasta ahorita no he tenido ningún problema con alguna aplicación, tampoco algún bug ni problemas de compatibilidad.
Debo recalcar que lo actualice porque decía que mi equipo es compatible. Si el tuyo no lo es, obviamente no te lo recomiendo. De todas formas como usuario promedio, a mi parecer no te estás perdiendo de mucho. 🙂

**Comment by Dfredude** [comment]
### No
Windows 10 es el mejor OS que ha hecho microsoft hasta la fecha. Y sabemos por el historial de windows, que siempre la version sucesora de un buen Windows resulta ser una porqueria. Ejemplos:
- **_Windows XP_** > **_Windows Vista_**
- **_Windows 7_** > **_Windows 8_**
- **_Windows 10_** > **_Windows 11_**
